README.md

```diff
@@ -10,6 +10,8 @@
 
 # Robot Framework AIO  <!-- omit in toc -->
 
+[![License: BIOSL v4](http://bios.intranet.bosch.com/bioslv4-badge.svg)](#license)
```

Comment:
Hi Nguyen,
Licence is wrong. Please correct.
Thank you,
 Thomas

Reply:
Hi Thomas,

I have update the README accordingly in commit https://github.com/test-fullautomation/RobotFramework_AIO/pull/16/commits/5ef5c433edbc638fd95f2818ec6b7eda81744133

Thank you,
Ngoan